drivers/net/macb.c

```diff
@@ -1325,6 +1408,18 @@ static int macb_eth_of_to_plat(struct udevice *dev)
 	if (!pdata->iobase)
 		return -EINVAL;
 
+	if (IS_ENABLED(CONFIG_GPIO)) {
+		/* optional PHY reset-related properties */
+		ret = gpio_request_by_name(dev, "phy-reset-gpios", 0, &macb->phy_reset_gpio,
```

Comment:
if it's optional you have to check for -ENOENT and dance from here on.

Reply:
+